### ISSUE TITLE
Pour lundi 27 mars 2017.

### ISSUE THREAD

**Issue by ThomasGire** [EdisonLorgues1SD1617/Devoirs#55]
1. Exercices [1 à 3 page 214](https://raw.githubusercontent.com/EdisonLorgues1SD1617/Math1SD1617/master/Donn%C3%A9es/Chapitres/7.%20Poduit%20scalaire/Images/1-3p214.png).

1. Lire et questionner la proposition 16 du [polycopié](https://github.com/EdisonLorgues1SD1617/Math1SD1617/blob/master/Donn%C3%A9es/Chapitres/7.%20Poduit%20scalaire/Polycopie/ProduitScalaire.pdf) sur le produit scalaire: Règles de calcusl avec le produit scalaire.

@EdisonLorgues1SD1617/classe

- [ ] Alexandre Bouille
- [ ] Allan Hourdin
- [ ] Antoni Gagno
- [ ] Aurore Charles
- [ ] Bassir El Ouaadi
- [ ] Camille Lecour
- [ ] Cédric Prast
- [ ] Eliot Gonnet
- [ ] Fabien Viazzi
- [ ] Gabin Salucci-Battaglia
- [ ] Grégory Faitot
- [ ] Guillaume Delattre
- [ ] Guillaume Raynaud
- [ ] Gwendal Jean
- [ ] Henry Jugy
- [ ] Kellian Costa
- [ ] Louis Hassler
- [ ] Louis Vilain
- [ ] Ludovic Nunes Da Silva
- [ ] Marion Deisard-Decot
- [ ] Mary Michel
- [ ] Mathilde Lacroix
- [ ] Maximilian Riepl
- [ ] Mohamed Ayadi
- [ ] Nicolas Doussin
- [ ] Noe Conti
- [ ] Olivier Piollet
- [ ] Thomas Slack
- [ ] Virgil Couturier
- [ ] William Fernandes
- [ ] Yanis Galland

**Comment by MaryMichel** [comment]
https://github.com/MaryMichel/1-2-3p214

**Comment by MaryMichel** [comment]
Ff

**Comment by AllanHourdin** [comment]
@MaryMichel , je suis d'accord avec tes calculs et avec tes résultats ( je partagerai ce soir )

**Comment by WilliamFernandes1** [comment]
[ed.pdf](https://github.com/EdisonLorgues1SD1617/Devoirs/files/870765/ed.pdf)

**Comment by CedricPrast** [comment]
[exo math.txt](https://github.com/EdisonLorgues1SD1617/Devoirs/files/870877/exo.math.txt)

**Comment by MaximilianRiepl** [comment]
@MaryMichel j'ai vu dans ton 3e exercice que tes resultats en degres me semblent faux
@WilliamFernandes1 et @CedricPrast  je n'ai pas trouve les meme degres que vous
![img007](https://cloud.githubusercontent.com/assets/21678505/24334711/b3ef4b06-126f-11e7-8485-cb7c87c78f30.jpg)
![img009](https://cloud.githubusercontent.com/assets/21678505/24334712/b3f2a1c0-126f-11e7-83a1-d015539097bd.jpg)
